docs/tutorials/contracts/nfts/minting-nfts.md

```diff
@@ -0,0 +1,356 @@
+
+```bash
+export ID=YOUR_ACCOUNT_NAME
+```
+
+Test that the environment variable is set correctly by running:
+
+```bash
+echo $ID
+```
+
+Verify that the correct account ID is printed in the terminal. If everything looks correct you can now deploy your contract. 
+In the root of your NFT project run the following command to deploy your smart contract.
+
+```bash
+near deploy --wasmFile res/non_fungible_token.wasm --accountId $ID
+```
+
+<details>
+<summary>Example response: </summary>
+<p>
+
+```bash
+Starting deployment. Account id: ex-1.testnet, node: https://rpc.testnet.near.org, helper: https://helper.testnet.near.org, file: res/non_fungible_token.wasm
+Transaction Id E1AoeTjvuNbDDdNS9SqKfoWiZT95keFrRUmsB65fVZ52
+To see the transaction in the transaction explorer, please open this url in your browser
+https://explorer.testnet.near.org/transactions/E1AoeTjvuNbDDdNS9SqKfoWiZT95keFrRUmsB65fVZ52
+Done deploying to ex-1.testnet
+```
+
+</p>
+</details>
+
+> **Note:** For `mainnet` you will need to prepend your command with `NEAR_ENV=mainnet`. [See here](/docs/tools/near-cli#network-selection) for more information.
+
+### Minting your NFTs
+
+The NFT contract should be initialized before usage, but for now we'll initialize with the default metadata:
+
+```bash
+near call $ID new_default_meta '{"owner_id": "'$ID'"}' --accountId $ID
+```
+
+> **Tip:** you can find more info about the NFT metadata at [nomicon.io](https://nomicon.io/Standards/NonFungibleToken/Metadata.html).
+
+You can then view the metadata by running the following view call:
+
+```bash
+near view $ID nft_metadata
+```
+
+<details>
+<summary>Example response: </summary>
+<p>
+
+```json
+{
+  "spec": "nft-1.0.0",
+  "name": "Example NEAR non-fungible token",
+  "symbol": "EXAMPLE",
+  "icon": "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 288 288'%3E%3Cg id='l' data-name='l'%3E%3Cpath d='M187.58,79.81l-30.1,44.69a3.2,3.2,0,0,0,4.75,4.2L191.86,103a1.2,1.2,0,0,1,2,.91v80.46a1.2,1.2,0,0,1-2.12.77L102.18,77.93A15.35,15.35,0,0,0,90.47,72.5H87.34A15.34,15.34,0,0,0,72,87.84V201.16A15.34,15.34,0,0,0,87.34,216.5h0a15.35,15.35,0,0,0,13.08-7.31l30.1-44.69a3.2,3.2,0,0,0-4.75-4.2L96.14,186a1.2,1.2,0,0,1-2-.91V104.61a1.2,1.2,0,0,1,2.12-.77l89.55,107.23a15.35,15.35,0,0,0,11.71,5.43h3.13A15.34,15.34,0,0,0,216,201.16V87.84A15.34,15.34,0,0,0,200.66,72.5h0A15.35,15.35,0,0,0,187.58,79.81Z'/%3E%3C/g%3E%3C/svg%3E",
+  "base_uri": null,
+  "reference": null,
+  "reference_hash": null
+}
+```
+
+</p>
+</details>
+
+Now let's mint our first token! The following command will mint one copy of your NFT. Replace the `media` url with the one you [uploaded to IPFS](#uploading-the-image) earlier:
+
+```bash
+near call $ID nft_mint '{"token_id": "0", "receiver_id": "'$ID'", "token_metadata": { "title": "Some Art", "description": "My NFT media", "media": "https://bafkreiabag3ztnhe5pg7js4bj6sxuvkz3sdf76cjvcuqjoidvnfjz7vwrq.ipfs.dweb.link/", "copies": 1}}' --accountId $ID --deposit 10
```

Comment:
Why do we attach 10N to this call?

Reply:
I was following the instructions from https://examples.near.org/NFT 
I assume that setting 10N is a way to avoid any issue due to insufficient gas, but I don't really know. 

Should we use a different value?

Reply:
Updated to 0.1 N